api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentTooltipTemplate.md

```diff
@@ -29,6 +29,399 @@ A template name or container.
+        computed: {
+            scheduler: function() {
+                return this.$refs[schedulerRefKey].instance; 
+            }
+        }
+    };
+    </script>
+
+    <!-- tab: Template.vue -->
+    <template>
+        <div class="dx-tooltip-appointment-item">
+            <div class="dx-tooltip-appointment-item-marker">
+                <div
+                    class="dx-tooltip-appointment-item-marker-body"
+                    :style="{backgroundColor: markerColor}"
+                ></div>
+            </div>
+            <div class="dx-tooltip-appointment-item-content">
+                <div class="dx-tooltip-appointment-item-content">
+                    <div class="dx-tooltip-appointment-item-content-subject">
+                        {{data.appointmentData.text}}
+                    </div>
+                    <div class="dx-tooltip-appointment-item-content-date">
+                        {{data.appointmentData.startDate.toString()}}
+                    </div>
+                </div>
+            </div>
+            <div v-if="isDeleteButtonExist" class="dx-tooltip-appointment-item-delete-button-container">
+                <DxButton
+                    class="dx-tooltip-appointment-item-delete-button"
+                    icon="trash"
+                    stylingMode="text"
+                    @click="onClick"
+                ></DxButton>
+            </div>
+        </div>
+    </template>
+
+    <script>
+    import {DxButton} from 'devextreme-vue/button';
+    import { employees } from './data.js';
+
+    export default {
+        components: {
+            DxButton,
+        },
+        data() {
+            return {
+                employees,
+            };
+        },
+        methods: {
+            onClick(e){
+                this.$emit('delete-button-click', e);
+            },
+        },
+        props: {
+            data: {
+                type: Object,
+                default: () => {},
+            },
+            markerColor: {
+                type: String
+            },
+            isDeleteButtonExist: {
+                type: Boolean
+            }
+        },
+    };
+    </script>
+
+
+##### React
+
+    <!-- tab: App.js -->
+    import React, { useCallback } from 'react';
+
+    import 'devextreme/dist/css/dx.light.css';
+
+    import Scheduler from 'devextreme-react/scheduler';
+
+    import Tooltip from "./Tooltip.js";
+
+    const App = () => {
+        const schedulerRef =  useRef(null);
+
+        const appointmentTooltip = useCallback((props) => {
+            const scheduler = schedulerRef.current;
+
+            const getDisabled = (employeeID) => employees.find((employee) => {
+                return employee.id === employeeID;
+            }).disabled;
+
+            const getColor = (employeeID) => employees.find((employee) => {
+                return employee.id === employeeID;
+            }).color;
+
+            const color = getColor(props.appointmentData.employeeID);
```

Comment:
Let's change it to:
```suggestion
            // NOTE: You can get color from resources with props.appointmentData.resouceId
            const color = '#337ab7';
```

Please check the text in the NOTE comment, maybe we can write this sentence better :)